packages/mobile/src/app/DappKitAccountScreen.tsx

```diff
@@ -0,0 +1,77 @@
+import FullscreenCTA from '@celo/react-components/components/FullscreenCTA'
+import { componentStyles } from '@celo/react-components/styles/styles'
+import {
+  AccountAuthRequest,
+  AccountAuthResponseSuccess,
+  produceResponseDeeplink,
+} from '@celo/utils/src/dappkit'
+import * as React from 'react'
+import { withNamespaces, WithNamespaces } from 'react-i18next'
+import { Linking, Text, View } from 'react-native'
+import { NavigationParams, NavigationScreenProp } from 'react-navigation'
+import { connect } from 'react-redux'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { RootState } from 'src/redux/reducers'
+
+interface OwnProps {
+  account: string
+  errorMessage?: string
+  navigation?: NavigationScreenProp<NavigationParams>
+}
+
+type Props = OwnProps & WithNamespaces
+
+class DappKitAccountAuthScreen extends React.Component<Props> {
+  static navigationOptions = { header: null }
+
+  getErrorMessage() {
+    return (
+      this.props.errorMessage ||
+      (this.props.navigation && this.props.navigation.getParam('errorMessage')) ||
+      'Yes!'
+    )
+  }
+
+  linkBack = () => {
+    if (!this.props.navigation) {
+      return
+    }
+
+    const request: AccountAuthRequest = this.props.navigation.getParam('dappKitRequest', null)
+
+    if (request === null) {
+      return
+    }
+
+    navigate(Screens.WalletHome)
```

Comment:
Why navigate just before opening a url anyway?

Reply:
That was me trying to do https://github.com/celo-org/celo-monorepo/issues/354. Basically I didn't want to leave the auth screen up when a user returns to the wallet, but assume whoever works on #354 will do this properly